src/universal/containers/Action/ActionContainer.js

```diff
@@ -17,6 +23,8 @@ const updateAnalyticsPage = (dispatch, lastPage, nextPage) => {
 };
 
 @connect()
+@withRouter
+@socketWithPresence
```

Comment:
i think we want to avoid setting up a socket connection for _every_ client, rather only those that have logged in. pheraps move the logic from ActionContainer to socketWithPresence?